.github/workflows/timeseries-build-pull-request.yml

```diff
@@ -114,6 +114,31 @@ jobs:
       with:
         name: unit-test-report
         path: /tmp/report.txt
+
+  timeseries-automation-test:
+    name: Timeseries Automation Test
+    permissions:
+      contents: read
+      packages: read          # needed for actions/checkout
+    runs-on: ubuntu-24.04
+    steps:
+    - uses: actions/checkout@v1
+      with:
+          path: timeseries
+          persist-credentials: false
+    - name: Run time-series-analytics-microservices automation tests
+      run: |
+        cd "${{ github.workspace }}"
+        cd ./microservices/time-series-analytics/functional

```

Comment:
Synced with @pooja-intel , we are keeping this folder as "tests-functional".
If we keep it as tests/functional then we have ignore it while running unit tests etc.,